tools/torch/trace_torchvision.py

```diff
@@ -336,11 +336,15 @@ def get_detection_input(batch_size=1, img_width=224, img_height=224):
         # remove extension
         filename = filename[:-3] + ".onnx"
         example = get_image_input(args.batch_size, args.img_width, args.img_height) 
+
+        # change for detection
+        out_name = "prob"
```

Comment:
Do we want to pass it as argument to the script, just in case ? and have the default value as `prob` ?

Reply:
Good ideea, I'll do that